restructuredtext_lint/lint.py

```diff
@@ -3,15 +3,10 @@
 from docutils.core import Publisher
 from docutils.nodes import Element
 
+from restructuredtext_lint import ignores
```

Comment:
Please use singular form for consistency within the repo. We don't call this file `lints` after all =P